src/Collectors/CommandCollector.php

```diff
@@ -47,7 +47,7 @@ public function registerEventListeners(): void
                 if ($transaction) {
                     $this->stopTransaction(
                         $transaction_name,
-                        $event->exitCode === 0 ? 200 : 500
+                        0 === $event->exitCode ? 200 : 500
```

Comment:
Are you not interested in the real exit code value? I think it might be useful for debugging purposes.

```suggestion
                        $event->exitCode
```

Reply:
I'm starting to believe that GH code suggestions are not the best idea, they only fix an isolated part of the code.